src/client/thirdParty/Inkeep.tsx

```diff
@@ -31,12 +33,24 @@ const useInkeepSettings = (): InkeepSharedSettings => {
   const integrationId = env.NEXT_PUBLIC_INKEEP_INTEGRATION_ID;
   const organizationId = env.NEXT_PUBLIC_INKEEP_ORG_ID;
 
+  const { data } = useVisitorData({ extendedResult: true, timeout: FPJS_CLIENT_TIMEOUT });
+  const visitorId = data?.visitorId || '';
+
+  const logEventCallback = (event: any) => {
+    debugger;
```

Comment:
Left-over debugger statement?

Reply:
Whoops, yeah. I was testing it and forgot to remove

Reply:
Removed